spinn_front_end_common/utilities/globals_variables.py

```diff
@@ -13,6 +13,17 @@ def get_simulator():
     return _simulator
 
 
+def get_not_running_simulator():
+    global _simulator, _failed_state
```

Comment:
It looks like this just does the same as get_simulator, with an additional check to verify that the simulator isn't running.  Can this instead call get_simulator rather than repeating the code?